packages/contracts-periphery/contracts/universal/drippie/dripchecks/CheckBalanceHigh.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import { IDripCheck } from "../IDripCheck.sol";
+
+contract CheckBalanceHigh is IDripCheck {
+    struct Params {
+        uint256 threshold;
+    }
+
+    function encode(Params memory _params) external pure {}
```

Comment:
Note here: https://github.com/ethereum-optimism/optimism/blob/2f8c302f072001ac27919ad0b2887a985fa4a445/packages/contracts-periphery/contracts/universal/drippie/IDripCheck.sol#L5-L8

And see here: https://github.com/ethereum-optimism/optimism/blob/2f8c302f072001ac27919ad0b2887a985fa4a445/packages/contracts-periphery/src/drippie-config.ts#L43-L51

To better understand why this exists.